TriblerGUI/debug_window.py

```diff
@@ -270,15 +270,18 @@ def load_open_files_tab(self):
         self.window().open_files_tree_widget.clear()
         gui_item = QTreeWidgetItem(self.window().open_files_tree_widget)
 
-        open_files = my_process.open_files()
-        gui_item.setText(0, "GUI (%d)" % len(open_files))
-        self.window().open_files_tree_widget.addTopLevelItem(gui_item)
-
-        for open_file in open_files:
-            item = QTreeWidgetItem()
-            item.setText(0, open_file.path)
-            item.setText(1, "%d" % open_file.fd)
-            gui_item.addChild(item)
+        try:
+            open_files = my_process.open_files()
+            gui_item.setText(0, "GUI (%d)" % len(open_files))
+            self.window().open_files_tree_widget.addTopLevelItem(gui_item)
+
+            for open_file in open_files:
+                item = QTreeWidgetItem()
+                item.setText(0, open_file.path)
+                item.setText(1, "%d" % open_file.fd)
+                gui_item.addChild(item)
+        except psutil.AccessDenied as exc:
+            gui_item.setText(0, "Unable to get open files for GUI (%s)" % exc)
```

Comment:
Doesn't this need a `self.window().open_files_tree_widget.addTopLevelItem(gui_item)`?